e2e/test/e2e-tx-serial-transfer.test.ts

```diff
@@ -76,4 +79,18 @@ describe("Transaction: serial transfer", () => {
     it("Receiver nonce not increased", async () => {
         expect(await send("eth_getTransactionCount", [BOB, "latest"])).eq(ZERO);
     });
+    it("Receiver balance is increased", async () => {
+        expect(await sendGetBalance(BOB)).eq(parseInt(TEST_BALANCE, 16) + TEST_TRANSFER)
+    });
+    it("Sender balance is decreased", async () => {
+        expect(await sendGetBalance(ALICE)).eq(parseInt(TEST_BALANCE, 16) - TEST_TRANSFER)
+    });
+    it("Send transaction to new account", async () => {
```

Comment:
Named accounts represents existing accounts.
Use a random address to represent a new account that does not exist.
There is already a function that generates random accounts in the RPC module.